sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/NotificationHookLiveTests.cs

```diff
@@ -343,7 +341,7 @@ public async Task UpdateEmailNotificationHookWithEveryMemberAndNewInstance()
         }
 
         [RecordedTest]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21177")]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/21504")]
```

Comment:
We fixed the Update implementation, but we have other issue when testing webhooks (https://github.com/Azure/azure-sdk-for-net/issues/21504).